app/controllers/assessments_controller.rb

```diff
@@ -145,6 +145,8 @@ def create
       # Determine if a user created this assessment or a monitoree
       @assessment.who_reported = current_user.nil? ? 'Monitoree' : current_user.email
 
+      @assessment.reported_at = DateTime.strptime(params[:reported_at], '%Y-%m-%d %H:%M') unless params.permit(:reported_at)[:reported_at].nil?
```

Comment:
For your first comment, we don't need to do that, because if `reported_at` is not set in the request, it will be automatically set to the same value as `created_at` (see `assessment.rb` change).

As for the second comment, I'll take another look at what additional checks we might want to do.